buidler-app/package.json

```diff
@@ -1,32 +1,41 @@
 {
-  "name": "test123",
+  "name": "@aragon/apps-payroll",
   "version": "0.0.1",
-  "description": "placeholder-app-description",
+  "description": "",
   "scripts": {
-    "postinstall": "npm run build-app",
-    "build-app": "cd app && npm install && cd ..",
-    "compile": "buidler compile --force",
+    "postinstall": "npm run compile",
     "start": "buidler start",
-    "test": "buidler test --network buidlerevm"
+    "console": "buidler console",
+    "compile": "buidler compile --force",
+    "devchain": "buidler node",
+    "build": "cd app && npm run build",
+    "lint": "buidler check && echo 'Lint complete.'",
+    "test": "npm run ganache-cli:test",
+    "test:gas": "GAS_REPORTER=true npm run ganache-cli:test",
+    "coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:test",
+    "ganache-cli:test": "./node_modules/@aragon/test-helpers/ganache-cli.sh",
+    "apm:publish:major": "buidler publish major",
+    "apm:publish:minor": "buidler publish minor",
+    "apm:publish:patch": "buidler publish patch"
   },
   "dependencies": {
-    "@aragon/abis": "^1.1.0",
     "@aragon/apps-finance": "^3.0.0",
-    "@aragon/apps-shared-migrations": "^1.0.0",
-    "@aragon/apps-shared-minime": "^1.0.2",
     "@aragon/apps-token-manager": "^2.1.0",
     "@aragon/apps-vault": "^4.1.0",
-    "@aragon/os": "^4.3.0",
-    "@aragon/ppf-contracts": "^1.2.0"
+    "@aragon/minime": "^1.0.0",
+    "@aragon/os": "^4.3.0"
   },
   "devDependencies": {
     "@aragon/buidler-aragon": "^0.2.0",
     "@aragon/test-helpers": "^2.1.0",
     "@nomiclabs/buidler": "^1.0.2",
     "@nomiclabs/buidler-etherscan": "^1.2.0",
+    "@nomiclabs/buidler-solhint": "^1.2.0",
     "@nomiclabs/buidler-truffle5": "^1.1.2",
     "@nomiclabs/buidler-web3": "^1.1.2",
-    "bignumber.js": "^9.0.0",
+    "buidler-gas-reporter": "^0.1.3",
```

Comment:
Packages to support gas report and test coverage once we release `test-helpers` web3 1.x compatible.